template/config/file.sls

```diff
@@ -21,3 +23,5 @@ template-config:
     - user: root
```

Comment:
Should `user` and `group` be configurable as general rule?   => future PR!!

Reply:
We're trying to move away from pillar values as much as possible, so we need to replace these with one of the alternatives being looked into.  So leaving it for now, at least.

Reply:
Cool. sounds reasonable.